addons/base_import/static/src/js/import_action.js

```diff
@@ -168,6 +172,7 @@ var DataImport = AbstractAction.extend({
         this.setup_separator_picker();
         this.setup_float_format_picker();
         this.setup_date_format_picker();
+        this.setup_sheets_picker();
```

Comment:
This may be a bad idea? The sheet picker *needs* remote data to be properly set up. Other pickers set up here have completely static inputs. I'm not sure select2 can just be re-invoked / re-setup on an input either. It probably needs to either be destroyed and recreated, or needs to be *updated* rather than reinvoked.